src/components/ProvideDetails.jsx

```diff
@@ -148,11 +148,11 @@ const provideDetails = props => {
 			rest.formik.errors.location = errorsReturned;
 		}
 		else {
-			const addressParts = props.formik.values.location.split(',');
+			const addressParts = location.split(',');
 			rest.formik.setFieldValue('requestTypeDescription', describeTheProblem);
 			rest.formik.setFieldValue('subRequestTypeAddress', addressParts[0]);
 			rest.formik.setFieldValue('subRequestTypeCity', addressParts[1]);
-			rest.formik.setFieldValue('subRequestTypeZip', addressParts[2]);
+			rest.formik.setFieldValue('subRequestTypeZip', (addressParts.length === 3) ? addressParts[2] : addressParts[3]);
```

Comment:
not as part of this pr, but you could probably make a utility for this, or address helpers if we have a bunch fo address stuff. This way we can add tests.

Reply:
not blocking.